app/services/we_work_remotely.rb

```diff
@@ -0,0 +1,83 @@
+class WeWorkRemotely < JobFetcher
+  BASE_URI = "https://weworkremotely.com/categories/2-programming/jobs.rss"
+
+  DIVS_OR_LIS        = /<\/div>|<li>/
+  NONBREAKING_SPACES = /&nbsp;/
+  NEWLINES_OR_TABS   = /\n+|\t+/
+  ESCAPED_QUOTES     = "\""
+
+  def self.scrape
+    technologies = Technology.pluck(:name)
+    feed = self.pull_feed
```

Comment:
It would be nice to have a test for what happens if WeWorkRemote goes down when we run the scraper